ui/package.json

```diff
@@ -1,55 +1,54 @@
 {
-  "name": "ui",
+  "name": "@hcmi-portal/ui",
   "private": true,
   "type": "module",
   "version": "1.0.0",
   "scripts": {
-    "build": "react-router build",
+    "build": "vite build",
     "dev": "vite",
     "start": "react-router-serve ./build/server/index.js",
     "typecheck": "react-router typegen && tsc"
   },
   "dependencies": {
     "@emotion/css": "^11.13.5",
     "@emotion/react": "^11.14.0",
-    "@nivo/bar": "^0.99.0",
-    "@nivo/pie": "^0.99.0",
+    "@emotion/styled": "^11.14.1",
+    "@nivo/bar": "^0.52.0",
+    "@nivo/pie": "^0.52.0",
     "@overture-stack/arranger-components": "^3.0.0",
     "@react-oauth/google": "^0.12.2",
-    "@react-router/node": "^7.7.1",
-    "@react-router/serve": "^7.7.1",
```

Comment:
I created the new Vite project trying out the react-router framework. I ended up removing this and reverting to a newer version of the classic react router.